src/main/java/org/broadinstitute/hellbender/utils/SimpleInterval.java

```diff
@@ -186,17 +186,17 @@ public final int size() {
     }
 
     /**
-     * Determines whether this interval overlaps the provided interval.
+     * Determines whether this interval overlaps the provided locatable.
      *
      * @param other interval to check
      * @return true if this interval overlaps other, otherwise false
      */
-    public final boolean overlaps( final SimpleInterval other ) {
```

Comment:
Thanks for generalizing these methods. I had miss-remembered them as already working on `Locatable` directly.